package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "andelaweek3optionalChallenge",
+  "version": "1.0.0",
+  "description": "",
+  "main": "missingNumbers.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/SeR4h/andelaweek3optionalChallenge.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/SeR4h/andelaweek3optionalChallenge/issues"
+  },
+  "homepage": "https://github.com/SeR4h/andelaweek3optionalChallenge#readme",
+  "dependencies": {},
+  "devDependencies": {
+    "jest": "^24.9.0"
+  }
+}
```

Comment:
Add an end of line at the end of this file. Just autoformat the code using your text editor